src/umode.rs

```diff
@@ -300,7 +300,18 @@ impl UmodeTask {
         Ok(())
     }
 
-    fn execute_request<T: DataInit>(
+    /// Execute a NOP request.
+    pub fn execute_nop() -> Result<(), Error> {
+        Self::execute_request::<u8>(UmodeRequest::Nop, None)?;
```

Comment:
I forgot to mention this before (and it's not really related to this patch), but the need for a dummy generic here is kind of silly. I guess that's because `T` must be `DataInit`, and the unit type isn't `DataInit`?

Reply:
It's silly and annoying, agree. Option None needs a type, and that type must be DataInit. Any suggestion on how to get around this in a way that feels acceptable is okay. Whenever I meet an idiotic rust stance I go for the pragmatic way.